pkg/quarterdeck/quarterdeck.go

```diff
@@ -232,6 +232,12 @@ func (s *Server) Routes(router *gin.Engine) (err error) {
 			users.GET("", middleware.Authorize(perms.ReadCollaborators), s.UserList)
 			users.DELETE("/:id", middleware.Authorize(perms.RemoveCollaborators), s.UserDelete)
 		}
+
+		// Accounts Resource - endpoint for users to manage their own account
+		accounts := v1.Group("/accounts", authenticate)
+		{
+			accounts.PUT("/:id", middleware.Authorize(perms.ReadCollaborators), s.AccountUpdate)
```

Comment:
I think this endpoint should actually require no permissions; any user should be able to update their own account via this  endpoint, and the check you have should prevent users from updating accounts of other users.